indico/modules/events/controllers/admin.py

```diff
@@ -149,3 +152,20 @@ def _process(self):
             flash(_('Settings have been saved'), 'success')
             return redirect(url_for('events.unlisted_events'))
         return WPEventAdmin.render_template('admin/unlisted_events.html', 'unlisted_events', form=form)
+
+
+class RHAutoLinker(RHAdminBase):
+    """Manage patterns which are automatically linked in notes."""
+
+    def _process(self):
+        return WPEventAdmin.render_template('admin/autolinker.html')
```

Comment:
```suggestion
        return WPEventAdmin.render_template('admin/autolinker.html', 'autolinker')
```

to mark the sidemenu item as active